README.md

```diff
@@ -1,41 +1,34 @@
 # Chicken Tinder
-
 This is the monorepo for Chicken Tinder.
-Use `npm run format` and `npm run lint` before commiting/pushing.
-
-## Product Vision
-
-For friend groups who can’t decide where to eat in a timely manner, Chicken Tinder is a web app that collects food preferences within groups and picks a restaurant catered to the group’s preferences. Unlike Yelp, our product enables collaboration within customizable groups of users.
-
-## User Interface Prototypes
-
-[in Figma](https://www.figma.com/file/NYJDxRDX5nzyrkO2d03S7I/Chicken-Tinder?type=design&node-id=0%3A1&mode=design&t=a2ww8RXXaC2aStSE-1)
-
-## Configure the repo
 
-Run `npm install` to install dependencies.
-Run `npm run backend` and `npm run frontend` to start up servers locally.
+Product Vision: For friend groups who can’t decide where to eat in a timely manner, Chicken Tinder is a web app that collects food preferences within groups and picks a restaurant catered to the group’s preferences. Unlike Yelp, our product enables collaboration within customizable groups of users.
 
-## Configure the backend
+## Artifacts
+UI prototypes and designs are located [in Figma](https://www.figma.com/file/NYJDxRDX5nzyrkO2d03S7I/Chicken-Tinder?type=design&node-id=0%3A1&mode=design&t=a2ww8RXXaC2aStSE-1)
```

Comment:
Could you also add the following:
- link to the prototype: https://www.figma.com/proto/NYJDxRDX5nzyrkO2d03S7I/Chicken-Tinder?type=design&node-id=269-111&t=7F6iy4C3B3gihSNp-0&scaling=min-zoom&starting-point-node-id=269%3A111&show-proto-sidebar=1
- date that the UI was last updated: 2/27/24